packages/react-google-maps-api/src/components/dom/OverlayView.tsx

```diff
@@ -171,8 +171,8 @@ export class OverlayView extends React.PureComponent<
         this.containerElement
       )
     ) : (
-      <></>
-    )
+        <></>
```

Comment:
any idea why these extra spaces are being added? They give me a linting error

Reply:
Well, that something I wanted to tweak later as well. I think linter should not be used for styling, we are living in the age of Prettier. I could revert that from this PR if it bothers you too much though.

Reply:
Done. I've also added `prepublishOnly` script now to avoid the mess that happened earlier on Spectrum :)

Reply:
I am very much pro prettier. I've done a ton of work to get to the point where we are now. Before that me and @JustFly1984 kept having conflicting configurations.. 
Right now we're using prettier, but via eslint. And using eslint's autofix to format the code. Not sure why it doesn't work correctly in your env.

Reply:
I have a Prettier extension and it was reformatting every file I've touched, so I disabled it. Are you aware that you have Prettier disabled in Eslint config? Doesn't matter now, let's revisit that later.